docs/core/tools/dotnet-workload-update.md

```diff
@@ -0,0 +1,94 @@
+---
+title: dotnet workload update command
+description: The 'dotnet workload update' command updates installed workloads.
+ms.date: 07/08/2021
+---
+# dotnet workload update
+
+**This article applies to:** ✔️ .NET 6 Preview SDK and later versions
+
+## Name
+
+`dotnet workload update` - Updates installed workloads.
+
+## Synopsis
+
+```dotnetcli
+dotnet workload update
+    [--add-source <SOURCE>] [--configfile <FILE>]
+    [--disable-parallel] [--download-to-cache <CACHE>]
+    [--from-cache <CACHE>] [--from-previous-sdk]
+    [--ignore-failed-sources] [--include-previews]
+    [--interactive] [--no-cache] [--sdk-version <VERSION>] [--temp-dir <PATH>] [-v|--verbosity <LEVEL>]
+
+dotnet workload update -?|-h|--help
+```
+
+## Description
+
+The `dotnet workload update` command queries Nuget.org for updated workload manifests. It then updates local manifests, downloads new versions of the installed workloads, and removes all old versions of each workload.
```

Comment:
This exposes the implementation to the user right off the bat, can we preface it with a sentence like 'the command updates all installed workloads to the newest available versions'?